build.sbt

```diff
@@ -35,6 +37,9 @@ inThisBuild(
       )
     ),
     scalaVersion := scala213,
+    // TODO: remove when munit/scalacheck align on `test-interface` package
+    // version; it comes from `org.scala-native:sbt-scala-native` plugin
+    evictionErrorLevel := sbt.util.Level.Warn,
```

Comment:
this is the proposed solution to deal with test-interface-0.5.5 (scalacheck) vs 0.5.6 (munit) version mismatch.

Reply:
https://github.com/scalameta/munit-scalacheck/actions/runs/12940058255/job/36093532302?pr=64
```
[warn] found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[warn] 	* org.scala-native:test-interface_native0.5_2.12:0.5.6 (strict) is selected over 0.5.5
[warn] 	    +- org.scalameta:munit_native0.5_2.12:1.1.0           (depends on 0.5.6)
[warn] 	    +- org.scalacheck:scalacheck_native0.5_2.12:1.18.1    (depends on 0.5.5)
```

Reply:
I asked and it should be fine, it's binary compatible, but since it's early semver it might be more strict

Reply:
thanks for checking! also, nothing else here.